pyrelational/data/data_manager.py

```diff
@@ -205,6 +209,18 @@ def __getitem__(self, idx: int) -> Tuple[torch.Tensor]:
         # So that one can access samples by index directly
         return self.dataset[idx]
 
+    def _top_unlabelled_set(self, percentage: Optional[Union[int, float]] = None) -> None:
+        if percentage is None:
+            self.top_unlabelled = None
+        else:
+            if isinstance(percentage, int):
+                percentage /= 100
```

Comment:
I am afraid it might trigger some issues (for instance, someone trying different hit ratios from 0.1 to 1)

Reply:
I agree, I only hope this won't lead to any issue. :)